.github/pull_request_template.md

```diff
@@ -1,7 +1,8 @@
 ## Description
 
-## Checklist
+## [Checklist](https://github.com/stjude/proteinpaint/wiki/Pull-Request-Checklist)
 
-- [ ] Tests: added unit/integration tests and/or ran tests, or N/A.
-- [ ] Todos: added to code or documentation, or N/A.
-- [ ] Release notes: updated release.txt, prefixed a commit message with "fix:" or "feat:", or N/A.
+Check if you performed or verified to be not applicable.
```

Comment:
`Check if you performed` is not clear to me. I think you want to say something like':
Please make sure that each element in this checklist is accomplished or is not applicable.